README.md

```diff
@@ -118,6 +118,63 @@ This is ideal when you have your own workflows that you want to develop in addit
 
 For example, maybe you have your own existing CLI tools (e.g. using `terragrunt`). In this case, you may want to start by developing your own CLI.
 
+## Install
+
+### OSX
+
+From Homebrew directly
+
+```
+brew install atmos
+```
+
+### cloudposse/packages
+
+Using container
```

Comment:
don't think we need to limit it to containers only.
apk/apt will work in container and local hosts